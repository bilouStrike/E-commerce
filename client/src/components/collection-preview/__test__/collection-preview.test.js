import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import CollectionPreview from '../collection-preview.component';

it('expect to render cart item component', () => {
    const mockTile = 'hats';
    const mockItems = [{
            imageUrl: 'url',
            price: '250',
            name: 'name1',
            quantity: 2
        }
    ]
    expect(toJson(shallow(<CollectionPreview title={mockTile} items={mockItems}/>))).toMatchSnapshot();
});