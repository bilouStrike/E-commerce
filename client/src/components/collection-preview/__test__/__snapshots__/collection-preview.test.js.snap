// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render cart item component 1`] = `
<div
  className="collection-preview"
>
  <h1
    className="title"
  >
    HATS
  </h1>
  <div
    className="preview"
  >
    <Connect(CollectionItem)
      item={
        Object {
          "imageUrl": "url",
          "name": "name1",
          "price": "250",
          "quantity": 2,
        }
      }
    />
  </div>
</div>
`;
